deploy/linux-systemd/tigerbeetle-pre-start

```diff
@@ -0,0 +1,7 @@
+#!/bin/bash
+set -e
+set -o nounset
+
+if ! test -e ${TIGERBEETLE_DATA_FILE}; then
```

Comment:
Probably best to quote all of these, in case there's a space in any of them (most likely the data file).

Reply:
Thanks for catching this!